rcl/include/rcl/graph.h

```diff
@@ -440,6 +441,39 @@ rcl_get_node_names(
   rcutils_string_array_t * node_names,
   rcutils_string_array_t * node_namespaces);
 
+/// Return a list of available nodes in the ROS graph, including their security context names.
+/**
+ * \sa Works like rcl_get_node_names, but it also include the security context name the node is
```

Comment:
@ivanpauno `\sa` expects references as arguments.

Maybe you just want something like:
```suggestion
 * \sa An \ref rcl_get_node_names equivalent, but including in its output the security context name the node is using.
```
?

Reply:
Yeah, I was trying to do that. Thanks for the suggestion!

Reply:
See https://github.com/ros2/rcl/pull/515/commits/44a15be05484088c78a4e7cde08a021356757160